src/src/com/microsoft/aad/adal/UnexpectedServerResponseException.java

```diff
@@ -0,0 +1,14 @@
+package com.microsoft.aad.adal;
+
+import java.io.IOException;
+
+/**
+ * Created by imakhal on 2/17/2016.
+ */
+public class UnexpectedServerResponseException extends IOException {
```

Comment:
I agree completely.  That was a complete f$%^ up.  I think it would be worth changing this.  It is a breaking change, but it is a compile time breaking change rather than a runtime breaking change so it shouldn't change the behavior of existing apps.